docs/tutorials/first_app/blinky_compiler.md

```diff
@@ -1,70 +1,81 @@
+#### Desktop IDE
 
-### Selecting a target board
+To debug using a desktop IDE such as Keil uVision, IAR, or Eclipse, click the "Export" button under "Program Details", select your export platform and IDE and click "Export". Your browser will then download a zip file with the project files.
 
-The Mbed Online Compiler can build your application to match any Arm Mbed Enabled board. However, you have to select the target board before compiling.
+<span class="images">![](https://sarahmarshy.github.io/img/export.png)
+</span>
 
-#### Adding a board to your list
+#### Printf
 
-To add a board to your list, go to <a href="https://os.mbed.com/platforms/" target="_blank">the board's page on `os.mbed.com`</a>, and click the **`Add to your Mbed Compiler`** button:
+Another way to do basic debugging is to use the `printf` command in your code, then read the output using a serial terminal such as [PuTTY](http://www.putty.org/) or [CoolTerm](http://freeware.the-meiers.org/). For example, add `printf("Hello World!\n\r");` to the top of your main function, then recompile the program and flash it to your device.
 
-<span class="images">![](https://s3-us-west-2.amazonaws.com/mbed-os-docs-images/add_board.png)<span>Adding a board to the Mbed Online Compiler's board list</span></span>
+Unless otherwise specified, `printf` defaults to a baud rate/speed of `9600` on mbed OS. To determine which communication port your board is connected to, follow the instructions for your operating system below:
 
-#### Selecting a board
+##### Windows
 
-The compiler shows the current build board's name on the upper right corner:
+Open the Device Manager by pressing `Windows key + R`, type `devmgmt.msc` and click "OK." Under "Ports (COM & LPT)" your mbed board will be listed as a "USB Serial Device" next to its COM port.
 
-<span class="images">![](https://s3-us-west-2.amazonaws.com/mbed-os-docs-images/show_board.png)<span>Opening the list of boards</span></span>
+##### Linux
 
-Click the name to change your board as needed:
+Run `dmesg | grep tty` from your command line. 
 
-<span class="images">![](https://s3-us-west-2.amazonaws.com/mbed-os-docs-images/select_board.png)<span>Selecting a board</span></span>
+##### Mac
 
-### Compile and install
+Run `ls /dev/tty.*` from your command line. 
 
-The Mbed Online Compiler builds your program as a `.bin` file that you can install on your board.
 
-1. Click **Compile**.
 
-	<span class="images">![](https://s3-us-west-2.amazonaws.com/mbed-os-docs-images/compileandinstall.png)<span>The Compile menu; choose Compile to build and download your application</span></span>
+### Further Reading
 
-1. The program compiles:
+- Documentation
+	- [Mbed OS API's](https://os.mbed.com/docs/v5.6/reference/apis.html) - List of all API's available in Mbed OS
+	- [Peripheral Drivers](https://os.mbed.com/docs/v5.6/reference/drivers.html) - IO Driver API's (I2C, SPI, UART, ... etc)
```

Comment:
Again, why are we linking to the APIs and then linking to a subset of the APIs? They were included in the link above.

Reply:
see previous answer